scripts/import/dbSNP_v2/update_new_sets.pl

```diff
@@ -0,0 +1,337 @@
+}
+
+
+sub dump_new_variation_sets {
+  # this would dump the variation_sets table from the backup table and create a file vset_concat.txt with two columns which are var_id and sets of variation_set_id
+  my $dbhvar = shift;
+  my $tmp_num = shift;
+  my $chunk = shift;
+  my $size = shift;
+
+  my $start = $chunk * $tmp_num;
+  my $end = $chunk + $start;
+  $end = $end < $size ? $end : $size;
+
+  my $dump_vs = $dbhvar->prepare(qq{  SELECT variation_id, GROUP_CONCAT(DISTINCT(variation_set_id)) FROM variation_set_variation WHERE variation_id > $start AND variation_id <= $end GROUP BY variation_id});
+  open (FH, ">>$TMP_DIR/$tmp_vs_file" )
+      or die( "Cannot open $TMP_DIR/$tmp_vs_file: $!" );
+  $dump_vs->execute();
+
+  while ( my $aref = $dump_vs->fetchrow_arrayref() ) {
+    my @a = map {defined($_) ? $_ : '\N'} @$aref;
+    print FH join("\t", @a), "\n";
+  }
+  
+  $dump_vs->finish();
+  close FH;
+}
+
+
+sub dump_old_sql_variation_sets {
+  # this would dump old variation_sets table from the old database and create a file called variation_name_set.txt with two columns var_id and variation_set_id
+  my $dbhvar = shift;
+  my $tmp_num = shift;
+  my $chunk = shift;
+  my $size = shift;
+
+  my $start = $chunk * $tmp_num;
+  my $end = $chunk + $start;
+  $end = $end < $size ? $end : $size;
+
+
+  my $sql = qq{SELECT v.name, vs.variation_set_id from variation_set_variation vs LEFT JOIN variation v
+               ON v.variation_id = vs.variation_id where variation_set_id != 1 AND v.variation_id > $start AND v.variation_id <= $end  };
+  my $sth = $dbhvar->prepare($sql);
+
+  local *FH;
+  open (FH, ">>$TMP_DIR/$tmp_vset" )
+      or die( "Cannot open $TMP_DIR/$tmp_vset: $!" );
+
+  $sth->execute();
+
+  while ( my $aref = $sth->fetchrow_arrayref() ) {
+    my @a = map {defined($_) ? $_ : '\N'} @$aref;
+    print FH join("\t", @a), "\n";
+  }
+  
+  $sth->finish();
+  close FH;
+
+}
+ 
+sub dump_new_variation_feature {
+  # this would dump the variation_id and name from the variation_feature table creating a file called variation_feature.txt with two columns contains var_id and name
+  my $dbh = shift;
+  my $tmp_num = shift;
+  my $chunk = shift;
+  my $size = shift;
+
+  my $start = $chunk * $tmp_num;
+  my $end = $chunk + $start;
+  $end = $end < $size ? $end : $size;
+
+
+  my $sql = qq{SELECT DISTINCT(variation_id), variation_name from variation_feature where variation_id > $start AND variation_id <= $end };
+  my $sth = $dbh->prepare($sql);
+  
+  local *FH;
+  open (FH, ">>$TMP_DIR/$TMP_FILE" )
+      or die( "Cannot open $TMP_DIR/$TMP_FILE: $!" );
+
+  $sth->execute();
+
+  while ( my $aref = $sth->fetchrow_arrayref() ) {
+    my @a = map {defined($_) ? $_ : '\N'} @$aref;
+    print FH join("\t", @a), "\n";
+  }
+  
+  $sth->finish();
+  close FH;
+
+  
+}
+
+sub usage {
+
+  die "\n\tUsage: update_old_sets.pl -registry [registry file] -release [release number] \tOptional:  -tmp [temp folder] or gets set based on current directory 
```

Comment:
The name of the script is `update_new_sets.pl`
The option `-release` is only used if `-old_registry` is not defined. You could add a note about that.